diabetes_regression/training/training_dependencies.yml

```diff
@@ -0,0 +1,17 @@
+name: diabetes_training
+
+dependencies:
+
+  # The python interpreter version.
+- python=3.7.5
+
+- numpy
+- pandas
+- scikit-learn
+- tensorflow
```

Comment:
please remove unused dependencies tensorflow, keras, azure-storage-blob. I believe pipeline submit automatically installs the latest azureml defaults on remote compute in that case we specify the exact version we need

Reply:
Done (commented out keras and tf for ease of use)